core/module.h

```diff
@@ -349,6 +383,8 @@ class Module {
   std::vector<const ModuleTask *> visited_tasks_;
 
  protected:
+  std::atomic<int> overload_;
```

Comment:
Member variable comment.

Also, perhaps rename to children_overload_ or something like that.